tests/end_to_end/multi-switch/irix-o2-out.c

```diff
@@ -19,16 +19,12 @@ s32 test(s32 arg0) {
             // Duplicate return node #16. Try simplifying control flow for better match
             return (phi_a0_2 + 1) ^ phi_a0_2;
         }
-        if ((u32) (arg0 - 0x65) < 7U) {
-            switch (arg0) { // switch 1
-            case 107: // switch 1
-                phi_a0 = arg0 + 1;
-                // Duplicate return node #24. Try simplifying control flow for better match
-                D_410210 = phi_a0;
-                return 2;
-            }
-        } else {
-            goto block_23;
```

Comment:
This looks bad ("where did `goto block_23` go? shouldn't there be a `default:` somewhere?) -- but the `default: // switch 1` is down at line 70, unchanged.